biogeophys/FatesHydroWTFMod.F90

```diff
@@ -800,22 +831,32 @@ subroutine set_wrf_param_tfs(this,params_in)
 
     class(wrf_type_tfs)  :: this
     real(r8), intent(in) :: params_in(:)
+    real(r8) :: th_max
+    real(r8) :: th_min
 
     this%th_sat   = params_in(1)
     this%th_res   = params_in(2)
     this%pinot    = params_in(3)
     this%epsil    = params_in(4)
     this%rwc_ft   = params_in(5)
-    this%cap_corr = params_in(6)
-    this%cap_int  = params_in(7)
-    this%cap_slp  = params_in(8)
-    this%pmedia   = int(params_in(9))
+    this%psicap   = params_in(6)
```

Comment:
I think this is also unused, right? can this be removed?

Reply:
Also, the arguments initializing TFS parameterizations are not changed, so I don't think this call would work:

https://github.com/NGEET/fates/pull/737/files#diff-e5e2459b995e00503077bce86074962e2acc4780ca00f248242b153527655c42R5346-R5353

Reply:
removed psicap, resolved